cypress/e2e/features/treeCoverLossIndonesia.js

```diff
@@ -0,0 +1,17 @@
+import { When, Then } from '@badeball/cypress-cucumber-preprocessor';
+import {
+  goToDashboardFromHome,
+  selectCountryOnDashboard,
+} from '../../api/navigation-api';
+
+When('I visit Dashboard', () => {
+  goToDashboardFromHome();
+});
+
+When('I select Indonesia', () => {
+  selectCountryOnDashboard('Indonesia');
+});
```

Comment:
Here, we can use the dynamic country name from the scenario:
```javascript
When('I select {string}', (countryName) => {
  selectCountryOnDashboard(countryName);
});

Reply:
Nice! I'll update it soon!